packages/cli/src/commands/__tests__/commands.spec.ts

```diff
@@ -0,0 +1,113 @@
+import * as prismHttp from '@stoplight/prism-http';
+import * as yargs from 'yargs';
+import { createMultiProcessPrism, createSingleProcessPrism } from '../../util/createServer';
+import mockCommand from '../mock';
+import proxyCommand from '../proxy';
+
+const parser = yargs.command(mockCommand).command(proxyCommand);
+
+jest.mock('../../util/createServer', () => ({
+  createMultiProcessPrism: jest.fn().mockResolvedValue([]),
+  createSingleProcessPrism: jest.fn().mockResolvedValue([]),
+}));
+
+jest.spyOn(prismHttp, 'getHttpOperationsFromResource').mockResolvedValue([]);
```

Comment:
I'd transform this line into jest.mock() or the line above into jest.spyOn. Just to make it more perfect :)

Reply:
I cannot really do that — jest.mock won't make me mock the resolved value in a single statement, which I need.